pkgs/development/compilers/llvm/README.md

```diff
@@ -64,6 +64,21 @@
 
 ## Information on our current patch sets
 
+### Adding a patch
```

Comment:
Would it make more sense to put this _after_ the information about the current patch sets? Arguably it should be an independent second-level heading.

Reply:
Agreed.